package.json

```diff
@@ -27,7 +27,7 @@
     "release-stable": "babel-node ./tools/release-stable",
     "lint": "eslint src",
     "storybook": "start-storybook -p 9002 -c storybook",
-    "build-storybook": "build-storybook -c storybook -o storybook-static -s ./storybook/public",
```

Comment:
Just curious what removing `-s ./storybook/public` is doing

Reply:
I believe it is simply removing the directory since it is unused / isn't being populated with any static assets.

Reply:
Correct. With the latest storybook, it was failing because it couldn't read the directory.

Reply:
Great thanks for the explanation